logins-swift/Sync15Passwords/Sync15Logins.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,443 @@
+			);
+			path = Errors;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXHeadersBuildPhase section */
+		CE9D201D20914D0D00F1C8FA /* Headers */ = {
+			isa = PBXHeadersBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				CE9D202520914D0D00F1C8FA /* Sync15Logins.h in Headers */,
+				CDFA91EE2153568E0074CABF /* RustPasswordAPI.h in Headers */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXHeadersBuildPhase section */
+
+/* Begin PBXNativeTarget section */
+		CE9D201F20914D0D00F1C8FA /* Sync15Logins */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = CE9D202820914D0D00F1C8FA /* Build configuration list for PBXNativeTarget "Sync15Logins" */;
+			buildPhases = (
+				CEEA12EF20AA1D630018998A /* ShellScript */,
+				CE9D201B20914D0D00F1C8FA /* Sources */,
+				CE9D201C20914D0D00F1C8FA /* Frameworks */,
+				CE9D201D20914D0D00F1C8FA /* Headers */,
+				CE9D201E20914D0D00F1C8FA /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = Sync15Logins;
+			productName = Sync15Logins;
+			productReference = CE9D202020914D0D00F1C8FA /* Sync15Logins.framework */;
+			productType = "com.apple.product-type.framework";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		CE9D201720914D0D00F1C8FA /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastUpgradeCheck = 0930;
+				ORGANIZATIONNAME = Mozilla;
+				TargetAttributes = {
+					CE9D201F20914D0D00F1C8FA = {
+						CreatedOnToolsVersion = 9.3;
+						LastSwiftMigration = 0930;
+					};
+				};
+			};
+			buildConfigurationList = CE9D201A20914D0D00F1C8FA /* Build configuration list for PBXProject "Sync15Logins" */;
+			compatibilityVersion = "Xcode 10.0";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+			);
+			mainGroup = CE9D201620914D0D00F1C8FA;
+			productRefGroup = CE9D202120914D0D00F1C8FA /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				CE9D201F20914D0D00F1C8FA /* Sync15Logins */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		CE9D201E20914D0D00F1C8FA /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXShellScriptBuildPhase section */
+		CEEA12EF20AA1D630018998A /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+				"$(SRCROOT)/../../target/universal/release/liblogins_ffi.a",
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\ncd \"$SRCROOT\"/../../logins-sql/ffi\nenv -i PATH=$PATH OPENSSL_DIR=\"$SRCROOT\"/../../libs/ios/x86_64/openssl/ \\\n\"$HOME\"/.cargo/bin/cargo lipo --release --verbose\n";
```

Comment:
This still doesn't work. Trying to find the correct value for `OPENSSL_DIR` to get it to build.

Reply:
Can you try `OPENSSL_STATIC=1`?

Reply:
@eoger Thanks. If I compile pointing to the x86_64 directory I get `application-services/target/aarch64-apple-ios/release/liblogins_ffi.a(s2_meth.o) cputype (16777223) and cpusubtype (3) does not match previous archive members cputype (16777228) and cpusubtype (0) (all members must match)` and if I point to the universal directory I `application-services/libs/ios/universal/openssl/lib/libssl.a: File too small to be an archive`